src/main/java/org/cbio/gdcpipeline/writer/MutationWriter.java

```diff
@@ -64,10 +62,9 @@ public void writeHeader(Writer writer) throws IOException {
 
     private FieldExtractor<MutationRecord> createFieldExtractor(MutationRecord data) {
```

Comment:
Make this `AnnotatedRecord`, it's confusing ince you're calling `getHeaderWithAdditionalFields`, which isn't obvious that it would include the fields from an `AnnotatedRecord` since a casted MutationRecord is being passed in here.

Reply:
done